fastseq/optimizer/fairseq/beam_search_optimizer.py

```diff
@@ -459,7 +501,13 @@ def _generate(self,
         bsz = input_size[0]
         src_len = input_size[1]
         beam_size = self.beam_size
-        self.no_repeat_ngram_op = NGramRepeatBlock()
+        cuda_ngram_op_import = True
```

Comment:
initialize no_repeat_ngram_op  as None. In case of import exception, just pass. When no_repeat_ngram_op  is None, use cpu code. Otherwise, use gpu code. So we don't need to create new var cuda_ngram_op_import.

Reply:
Could we do this kind of checking (which kind of ops (e.g., cpu v.s. gpu) to use) inside the [ops implementation](https://github.com/microsoft/fastseq/blob/main/fastseq/ops/ngram_repeat_block.py#L34)? So that we do not need to do the similar check twice for fairseq and transformers. It will be easier for us to maintain and change the code in the future.